extensions/assetLibrary/src/client/Library.tsx

```diff
@@ -0,0 +1,466 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useState, Fragment, useEffect } from 'react';
+import formatMessage from 'format-message';
+import { Link, Pivot, PivotItem, Dialog, DialogType, Dropdown, MessageBar, MessageBarType, MessageBarButton, ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react';
+import { render, useHttpClient, useProjectApi, useApplicationApi } from '@bfc/extension-client';
+
+import { Toolbar, IToolbarItem } from '@bfc/ui-shared';
+
+import { ContentHeaderStyle, HeaderText, ContentStyle, contentEditor } from './styles';
+import { ImportDialog } from './importDialog';
+import { LibraryRef, LibraryList } from './libraryList';
+import { WorkingModal } from './workingModal';
+
+const DEFAULT_CATEGORY = formatMessage('Available');
+const RECENTLY_USED_KEY = 'recentlyUsedItems';
+
+const docsUrl = `https://aka.ms/composer-use-package-library`;
+
+const strings = {
+  title: formatMessage('Package Library'),
+  description: formatMessage('Discover and use components that can be installed into your bot.'),
+  descriptionLink: formatMessage('Learn more'),
+  installButton: formatMessage('Install Package'),
+  importDialogTitle: formatMessage('Install a Package'),
+  installProgress: formatMessage('Installing package...'),
+  recentlyUsedCategory: formatMessage('Recently Used'),
+  installedCategory: formatMessage('Installed'),
+  updateConfirmationPrompt: formatMessage('Any changes you made to this package will be lost! Are you sure you want to continue?'),
+  updateConfirmationTitle: formatMessage('Update Package'),
+  conflictConfirmationTitle: formatMessage('Conflicting changes detected'),
+  conflictConfirmationPrompt: formatMessage(
+    'This operation will overwrite changes made to previously imported files. Do you want to proceed?'
+  ),
+  removeConfirmationTitle: formatMessage('Remove Package'),
+  removeConfirmationPrompt: formatMessage(
+      'Any changes you made to this package will be lost! In addition, this may leave your bot in a broken state. Are you sure you want to continue?'
+  ),
+  requireEject: formatMessage('To install components, this project must have an ejected runtime. Please navigate to the runtime settings page.'),
+  ejectRuntime: formatMessage('Eject Runtime'),
+  noComponentsInstalled: formatMessage('No packages installed'),
+  noComponentsFound: formatMessage('No packages found. Check extension configuration.'),
+  browseHeader: formatMessage('Browse'),
+  installHeader: formatMessage('Installed'),
+  libraryError: formatMessage('Library Error'),
+  importError: formatMessage('Install Error'),
+}
```

Comment:
These need to be moved to where they are invoked at runtime rather than module load time.

Reply:
👍